### ISSUE TITLE
html render does not with custom csl file

### ISSUE THREAD

**Issue by ati-ozgur** [quarto-dev/quarto-cli#3740]
### Bug description

1. Create a new book project

`quarto create-project mybook --type book`

2. Update _quarto.yml to add csl file

`csl: ieee.csl`

3. try to render html and see the error

   `quarto render --to html`

4. Error: 

```
File C:_o-bug.csl not found in resource path
ERROR: File C:_o-bug.csl not found in resource path

``` 

This is the same error as #2945 

_No response_

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.

[mybook.zip](https://github.com/quarto-dev/quarto-cli/files/10290963/mybook.zip)

- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?

```
mybook $ quarto --version
1.2.280
```

- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

```
mybook $ systeminfo | findstr /B /C:"OS Name" /C:"OS Version"
OS Name:                   Microsoft Windows 10 Pr

o
OS Version:                10.0.19042 N/A Build 19042
```

**Comment by dragonstyle** [comment]
This should be fixed on the 1.3 pre-release builds...

**Comment by ati-ozgur** [comment]
I confirm that this bug is resolved in quarto version 1.3.56

**Comment by ati-ozgur** [comment]
it is resolved in pre-release builds of 1.3+
